slidedeck/src/slides/03-react-state/03-lifting-state-up.md

```diff
@@ -0,0 +1,40 @@
+import CodePlayground from '../../components/code-playground';
+
+<CodePlayground code={`
+class Name extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: ''
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(ev) {
+    const name = ev.target.value;
+    // TODO
```

Comment:
I'll do some "live coding" here just to show how to lift state up with props/callbacks.